### ISSUE TITLE
Confusion regarding :meta_key

### ISSUE THREAD

**Issue by jopotts** [rails-api/active_model_serializers#488]
Regarding this in the readme:

> If you would like to change the meta key name you can use the :meta_key option:

```
render json: @posts, serializer: CustomArraySerializer, meta: {total: 10}, meta_key: 'meta_object'
```

I think this is wrong. If the meta_key name is changed, then the key used for :meta should be the new name given.

So instead it should be this:

```
render json: @posts, serializer: CustomArraySerializer, meta_object: {total: 10}, meta_key: 'meta_object'
```

However, this still doesn't work as the initialize method in array_serializer.rb (and serializer.rb) has this:

```
@meta_key        = options[:meta_key] || :meta
@meta            = options[@meta_key]
```

So, either the code needs to change to be:

```
@meta_key        = options[:meta_key] || :meta
@meta            = options[@meta_key.to_sym]
```

Or the example needs to specify a symbol for the new :meta_key like this:

```
render json: @posts, serializer: CustomArraySerializer, meta_object: {total: 10}, meta_key: :meta_object
```

In summary, I think a code fix is needed and a doc update?
